operations.js

```diff
@@ -226,15 +226,26 @@ Operations.prototype.getPending = function getPending() {
     return self.pending;
 };
 
+Operations.prototype.getOutTombstone = function getOutTombstone(id) {
+    var self = this;
+
+    var op = self.requests.out[id] || null;
+    if (op && !op.isTombstone) {
+        return null;
+    }
+
+    return op;
+};
+
```

Comment:
nitpick: this is a bit of a weird pattern. Why not just have one `getOutReq` function that returns either a tombstone or a req, then check `.isTombstone` in `TChannelConnection#onCallErrorFrame`? 

I feel like this code obfuscates the flow for handling error frames when there's a tombstone.